torchvision/transforms/functional_tensor.py

```diff
@@ -758,6 +756,29 @@ def gaussian_blur(img: Tensor, kernel_size: List[int], sigma: List[float]) -> Te
     return img
 
 
+def gaussian_noise(img: Tensor, mean: float, sigma: float) -> Tensor:
+    if not (isinstance(img, torch.Tensor)):
+        raise TypeError(f"img should be Tensor. Got {type(img)}")
+
+    _assert_image_tensor(img)
+    dtype = img.dtype if torch.is_floating_point(img) else torch.float32
+    img, need_cast, need_squeeze, out_dtype = _cast_squeeze_in(
+        img,
+        [
+            dtype,
+        ],
+    )
+    # add the gaussian noise with the given mean and sigma.
+    normalize_img = img / 255.0
```

Comment:
We can't make this assumption that `img` range is [0, 255]. 
By default, we assume: [0, 1] for float images and [0, 255] for uint8 RGB images.
Let's not rescale image range and let user pick appropriate `mean` and `sigma` according to their data range.